content/add-a-printer-pop.md

```diff
@@ -94,14 +94,22 @@ Brother provides a driver installation tool for Linux users. Install the appropr
 
 ## Troubleshooting
 
+### Remove and Re-add the Printer in Settings
+
 If the printer stops working, open the <u>Printers</u> Application again and click the gear icon, and select **Remove Printer** to remove the printer and its settings. Repeat the steps above to add the printer and try printing the test page again. This process usually resolves many printing issues.
 
+### Check the Printer Status in CUPS
+
 To look at the CUPS (Common Unix Printing System) configuration and status window, please open this webpage:
 
 [localhost:631](http://localhost:631)
 
 The status window will show current print jobs, detected printers, and other information about the printing system. If you would like to share this printer with others on your local network, click on the 'Admin' link, under Server, click on the "Share printers connected to this system" and save the changes. Other computers on your network should than see that printer. When there is a prompt for your username and password, use your user name, and password used to login.
 
+### Assign a Static IP Address to the Printer
+
+Connectivity issues can occur when the printer is assigned an address using DHCP. Assigning a permanent, static IP address will resolve these issues. Reference your printer manufacturer's documentation for this process.
```

Comment:
A static IP address would either need to be from within the DHCP pool, or outside of the DHCP pool if the pool does not cover the entire subnet. Router configuration could be involved with determining and/or reserving an appropriate IP address. Do we think that is important enough to include somehow? (The user statically assigning whatever's currently dynamically assigned could result in e.g. an IP address conflict if the router later assigns that IP to someone else after the printer stops sending DHCP requests.)

Reply:
Yea that's a good point, I'll add that. I'll also mention that users in a corporate environment should reach out to their network admin for this process.